### PR TITLE
Fixes #23973: 

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/23973

### CODE REVIEW

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

File: policies/rudderc/src/ir/value.rs [hunk 1]
```diff
@@ -261,69 +377,124 @@ impl Expression {
...
+                                    x.push_str(" + @'\n.")
+                                }
+                            }
+                        };
+                    }
+                    nustache_render(&x)
                 }
+            },
+            Self::Scalar(s) => s.to_owned(),
+            //Target::Unix => s.to_string(),
```
Comment (amousset) [inline policies/rudderc/src/ir/value.rs:433]: why the commented code?

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR updated with a new commit

**Comment by Fdall** [comment]
PR rebased

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**